packages/react-components/src/accordion/docs/Accordion.stories.mdx

```diff
@@ -23,6 +25,28 @@ import { Text } from "@react-components/typography";
 
 ## Accessibility
 
+The content in the Accordion is collapsable. For that reason, it is important to ensure the navigation is not focusing on content that is hidden.
+
+Orbit is taking care of handling the [`aria-hidden`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute) value required to let the user know if the content is visible.
+
+<Preview>
+    <Story name="accessibility">
+        <Accordion>
+            <Item key="mars">
+                <Header as="h3">Mars</Header>
+                <Content>
+                    <Paragraph>Mars is the fourth planet from the Sun and the second-smallest planet in the Solar System.</Paragraph>
+                    <Button>Cutoff</Button>
```

Comment:
I am not sure why you add a example for the accessibility? This example seem similar to the default one.

Reply:
I wasn't sure when adding it, not a fan that this is the first example that you come across on the page. But the point was to give an example of a focusable item inside of the tab, not being focusable when collapsed. But I can remove that example.

Reply:
I think I would remove the example.